pkg/registry/core/pod/storage/eviction_test.go

```diff
@@ -60,6 +64,81 @@ func TestEviction(t *testing.T) {
 			eviction:      &policy.Eviction{ObjectMeta: metav1.ObjectMeta{Name: "foo", Namespace: "default"}, DeleteOptions: metav1.NewDeleteOptions(0)},
 			expectDeleted: true,
 		},
+		{
+			name: "matching pdbs with two disruptions allowed",
```

Comment:
I can't put an inline comment on the lines above but I think the tests with an array of a single PDB should probably have their names changed to indicate the single PDB case versus the multiple PDB case